### ISSUE TITLE
triplet loss fine tuning - error

### ISSUE THREAD

**Issue by vizakshat** [deepinsight/insightface#762]
Error in operator _minus0: [15:16:52] /work/mxnet/3rdparty/mshadow/../../src/operator/tensor/../elemwise_op_common.h:135: Check failed: assign(&dattr, vec.at(i)) Incompatible attr in node _minus0 at 1-th input: expected [10,512], got [11,512]

**Comment by leondgarse** [comment]
In case anyone still looking for this same issue, if you are using triplet for fine-tuning, the batch_size must be a number can be divided by **3**, like 24, 96, 120...
```sh
# My example
CUDA_VISIBLE_DEVICES='0,1' python -u train.py --network m1 --loss triplet --dataset emore --per-batch-size 96 --pretrained ./models/m1-arcface-emore/model --lr 0.0001
```
